### ISSUE TITLE
is there a wrong in loss?

### ISSUE THREAD

**Issue by xiaoxiaowannote** [IDKiro/CBDNet-pytorch#27]
may be your torch.lt is wrong

**Comment by Jaden-Yuan** [comment]
Which version of torch did you use?

**Comment by huntkao** [comment]
asym_loss = torch.mean(if_asym * torch.abs(0.3 - **torch.lt(gt_noise, est_noise**).float()) * torch.pow(est_noise - gt_noise, 2))
should be
asym_loss = torch.mean(if_asym * torch.abs(0.3 - **torch.lt(est_noise, gt_noise**).float()) * torch.pow(est_noise - gt_noise, 2)) ??

**Comment by MrBled** [comment]
> asym_loss = torch.mean(if_asym * torch.abs(0.3 - **torch.lt(gt_noise, est_noise**).float()) * torch.pow(est_noise - gt_noise, 2)) should be asym_loss = torch.mean(if_asym * torch.abs(0.3 - **torch.lt(est_noise, gt_noise**).float()) * torch.pow(est_noise - gt_noise, 2)) ??

This is an old issue but I have also confirmed this against the paper implementation.
